src/app/explanation/content.mdx

```diff
@@ -26,7 +26,7 @@ Může se však snando stát, že stabilních párování existuje pro danou sit
 
 Algoritmus pro nalezení nejlepšího párování pracuje v následujících krocích:
 
-1. Najdi všechna stabilní párování.
+1. Najdi všechna stabilní párování pomocí [rozšířeného Gale-Shapley algoritmu](https://en.wikipedia.org/wiki/Gale%E2%80%93Shapley_algorithm).
```

Comment:
Tady by to chtělo nastylovat odkaz, teď je modrou, která ti vypálí zrak 🧨

Reply:
Taky bych té stránce dala nějaký padding

Reply:
Ok, padding atd. jsem přidal. Jen bych potřeboval poradit s tou barvout odkazu.